src/functions/uploadToRSIS/application/data-mapper/cat-ce-mapper.ts

```diff
@@ -9,14 +8,22 @@ import {
   formatManoeuvreDangerous,
   optional,
   addIfSet,
+  formatQuestionSeriousCE,
+  formatQuestionFaultCE,
+  formatQuestionCompleted,
 } from './data-mapper';
+import { mapCommonCatCData } from './cat-c-common-mapper';
 
 export const mapCatCEData = (result: ResultUpload): DataField[] => {
   const t = result.testResult.testData;
 
-  const catCDataFields: DataField[] = mapCatCData(result);
+  const catCDataFields: DataField[] = mapCommonCatCData(result);
 
-  const m: DataField[] = [...catCDataFields,
+  const m: DataField[] = [
+    ...catCDataFields,
+    field('VEHICLE_CHECKS_COMPLETED', formatQuestionCompleted(t, 2)),
+    field('VEHICLE_CHECKS_TOTAL', formatQuestionFaultCE(t)),
+    field('VEHICLE_CHECKS_SERIOUS', formatQuestionSeriousCE(t)),
     field('UNCOUPLE_RECOUPLE_TOTAL', formatManoeuvreFault(t, 'uncoupleRecouple.fault')),
     field('UNCOUPLE_RECOUPLE_SERIOUS', formatManoeuvreSerious(t, 'uncoupleRecouple.fault')),
     field('UNCOUPLE_RECOUPLE_DANGEROUS', formatManoeuvreDangerous(t, 'uncoupleRecouple.fault')),
```

Comment:
UNCOUPLE_RECOUPLE_COMMENT - is this fixed separately to this?

Reply:
Separate ticket which is already in this branch :)